asapdiscovery-ml/asapdiscovery/ml/trainer.py

```diff
@@ -23,6 +28,20 @@
     ModelType,
     SchNetModelConfig,
 )
+
+try:
+    from mtenn.config import ViSNetModelConfig
+
+    # guard for visnet import. Refer MTENN issue #42
+    from mtenn.conversion_utils.visnet import HAS_VISNET
+except ImportError as e:
+    match = re.search(r"cannot import name 'ViSNetModelConfig'*", str(e))
+    if match:
+        warnings.warn("To use VisNet, pip install MTENN from Github", ImportWarning)
+        HAS_VISNET = False
+    else:
+        raise ImportError(f"Unexpected ImportError: {e}")
```

Comment:
With our current setup, we are testing against the official MTENN release (no `ViSNetModelConfig` implementation), and the github MTENN with `ViSNetModelConfig`. Ideally we want to elegantly fail on VisNet import and let the users of official MTENN carry on with a warning